src/components/Hearing.js

```diff
@@ -217,6 +219,21 @@ export class Hearing extends React.Component {
     this.setState({ showDeleteModal: false, commentToDelete: {} });
   }
 
+  getEyeTooltip() { // eslint-disable-line class-methods-use-this
+    const { formatMessage } = this.props.intl;
+    const openingTime = moment(this.props.hearing.open_at);
+    let text = <FormattedMessage id="eyeTooltip" />;
+    if (openingTime > moment()) {
```

Comment:
Here we should have `if (this.props.hearing.published && openingTime > moment())` so that drafts will get the correct eyeTooltip text.